exploration/outbreak_eda.ipynb

```diff
@@ -0,0 +1,273 @@
+    "    .drop(columns=[\"who_region\"])\n",
+    "    .apply(lambda column: column.str.upper().str.removesuffix(\"HEALTHDISTRICT\"))\n",
+    ")\n",
+    "\n",
+    "outbreaks_expanded_location_df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "outbreaks_admin2_df = (\n",
+    "    pd.concat(\n",
+    "        [\n",
+    "            outbreaks_expanded_location_df.drop(\"admin3\", axis=1),\n",
+    "            outbreaks_df.drop([\"who_region\", \"country\", \"location\"], axis=1),\n",
+    "        ],\n",
+    "        axis=1,\n",
+    "    )\n",
+    "    .query(\"spatial_scale == 'admin2'\")\n",
+    "    .sort_values(by=[\"ISO3\", \"admin1\", \"admin2\"])\n",
+    ")\n",
+    "\n",
+    "outbreaks_admin2_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We've cleaned up the outbreak dataset above. Now we'll import the administrative boundaries for all districts to give each of the district outbreaks a geographic context. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "district_geometries = gpd.read_file(\"geolocations.geojson\")\n",
+    "district_geometries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# sourcery skip: use-fstring-for-concatenation\n",
+    "yearly_cases_gdf = gpd.GeoDataFrame(\n",
+    "    (\n",
+    "        outbreaks_admin2_df.groupby(\n",
+    "            [\n",
+    "                \"start_year\",\n",
+    "                \"ISO3\",\n",
+    "                \"admin1\",\n",
+    "                \"admin2\",\n",
+    "                \"location_period_id\",\n",
+    "            ]\n",
+    "        )[\"total_suspected_cases\"]\n",
+    "        .sum()\n",
+    "        .reset_index()\n",
+    "    )\n",
+    "    .merge(district_geometries, on=\"location_period_id\")\n",
+    "    .assign(location=lambda gdf: gdf.admin2 + \", \" + gdf.admin1 + \", \" + gdf.ISO3)\n",
+    "    .set_index(\"location_period_id\")\n",
+    ")\n",
+    "\n",
+    "yearly_cases_gdf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will add a new column `outbreak` to represent presence of an outbreak in the district (outbreak = 1). This will be used to provide sums or counts of outbreaks and will later be used when we develop a machine learning model and need outbreak absence data points (outbreak = 0)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "yearly_cases_gdf[\"outbreak\"] = 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "yearly_cases_gdf"
+   ]
+  },
```

Comment:
Maybe we can merge this cell with the one above? Definitely a nit 😅

Reply:
Yes of course, it does look a little odd standing by itself there 😄